packages/sketch/src/commands/icons/generate.js

```diff
@@ -68,18 +68,19 @@ export function generate() {
             const [_type, _category, _subcategory, name, size] = parts;
             return name === icon && size === '32';
           });
-          const instance = symbol.createNewInstance();
-          instance.frame.offset(ICON_X_OFFSET, ICON_Y_OFFSET);
+          const instance = symbol?.createNewInstance();
```

Comment:
Oh wild, I think we might want to fail if the symbol doesn't exist. Maybe something like:

```js
if (!symbol) {
  throw new Error(`Unable to find symbol for icon ${icon}`);
}
```